zero_true/__init__.py

```diff
@@ -17,4 +17,7 @@
 from zt_backend.models.components.timer import Timer
 from zt_backend.models.components.iframe import iFrame
 from zt_backend.models.state.state import state
-from zt_backend.models.state.state import global_state
\ No newline at end of file
+from zt_backend.models.state.state import global_state
+from zt_backend.models.components.chat_ui import chat_ui
+import openai
+import logging
```

Comment:
These imports shouldn't be here.  If they are dependencies for this then they should be added to setup.cfg. Make sure to lock versions as we did with the other ones there